### PR TITLE
adding helmfile step

### PR DESCRIPTION
**Pull request by dustinvanbuskirk**
Adding step for helmfile

### CODE REVIEW

File: incubating/helmfile/step.yaml [hunk 1]
```diff
@@ -0,0 +1,236 @@
+kind: step-type
+version: '1.0'
+metadata:
+  name: dustinvanbuskirk/helmfile
```
Comment (olegsu) [inline incubating/helmfile/step.yaml]: The name cannot be under `dustinvanbuskirk` namespace in official steps, please change to `helmfile`

File: incubating/helmfile/step.yaml [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile test - test releases from state file (helm test)",
+                "default": ""
+            }
+        }
+    }
+  steps:
+    main:
+      name: helmfile
+      image: 'dustinvanbuskirk/cfstep-helmfile:${{HELMFILE_VERSION}}'
```
Comment (olegsu) [inline incubating/helmfile/step.yaml]: It is better to put the image under codefresh registry (https://hub.docker.com/r/codefresh)

File: incubating/helmfile/step.yaml (hunk 1 above)
Comment (dustinvanbuskirk) [reply incubating/helmfile/step.yaml]: Completed.

File: incubating/helmfile/step.yaml (hunk 2 above)
Comment (dustinvanbuskirk) [reply incubating/helmfile/step.yaml]: Completed

**Comment by olegsu** [comment]
LGTM

**Pull request merged**